basyx.aasenvironment/basyx.aasenvironment-http/src/test/java/org/eclipse/basyx/digitaltwin/aasenvironment/http/TestAasEnvironmentHTTP.java

```diff
@@ -36,61 +36,83 @@ public class TestAasEnvironmentHTTP {
 	private static final String ACCEPT_XML = "application/xml";
 	private static final String ACCEPT_AASX = "application/asset-administration-shell-package+xml";
 
-	private static ConfigurableApplicationContext appContext;
+	private ConfigurableApplicationContext appContext;
 
-	@BeforeClass
-	public static void startAasRepo() throws Exception {
+	@Before
```

Comment:
Why did you change this to @Before?

Reply:
Revert this to the previous state i.e. @BeforeClass